src/v2/guide/components.md

```diff
@@ -268,7 +268,7 @@ Vue.component('blog-post', {
 
 Now, whenever a new property is added to `post` objects, it will automatically be available inside `<blog-post>`.
 
-## Sending Messages to Parents with Events
+## Listening to Child Components Events
```

Comment:
Such a simple change and it's already easier to understand. Nicely done.

Reply:
Thank you Sarah!